darshan-util/pydarshan/darshan/cli/summary.py

```diff
@@ -378,6 +382,28 @@ def register_figures(self):
         io_cost_fig = ReportFigure(**io_cost_params)
         self.figures.append(io_cost_fig)
 
+        ################################
+        ## Per-Module Statistics
+        ################################
+        for mod in self.report.modules:
+            if mod in ["POSIX", "MPI-IO", "H5D"]:
+                if mod == "MPI-IO":
+                    com_acc_tbl_description = (
+                        "NOTE: MPI-IO accesses are given in "
+                        "terms of aggregate datatype size."
+                    )
+                else:
+                    com_acc_tbl_description = ""
+                com_acc_tbl_fig = ReportFigure(
+                    section_title=f"Per-Module Stats: {mod}",
```

Comment:
I've titled the new section "Per-Module Stats: MOD", but based on issue [#465](https://github.com/darshan-hpc/darshan/issues/465), maybe this should be "Per-Module Characteristics: MOD"?